src/Symfony/Component/Process/Process.php

```diff
@@ -231,7 +234,7 @@ public function __clone()
      * The STDOUT and STDERR are also available after the process is finished
      * via the getOutput() and getErrorOutput() methods.
      *
-     * @param callable|null $callback A PHP callback to run whenever there is some
+     * @param (callable('out'|'err', string):bool)|null $callback A PHP callback to run whenever there is some
```

Comment:
```suggestion
     * @param (callable('out'|'err', string):void)|null $callback A PHP callback to run whenever there is some
```

Same everywhere (except `waitUntil` which is a different API)